tdp/core/variables/cluster_variables.py

```diff
@@ -30,7 +30,7 @@ def __iter__(self):
     def initialize_cluster_variables(
         collections,
         tdp_vars,
-        override_folder=None,
+        override_folders=[],
```

Comment:
Ok if you want to put `()` since it is immutable but you must avoid mutable type in default see https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/. Use `None` and in the function you can do:
``` python
if override_folders is None:
  override_folders = []
```